app/models/vaccine.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Vaccine < ApplicationRecord
```

Comment:
Model looks great, could we move it and the migration into our mobile module? Apologies if I misled you on this, Jayson had a PR go in recently that added the code needed for migrations to work from there. 

New paths would be:
`modules/mobile/app/models/mobile/v0/vaccine.rb` and
`modules/mobile/db/migrate/20211112213456_create_vaccines.rb`

Reply:
Yes, and thank you for clearing this up. It wasn't clear to me if this should go in the base level or not. Given that there didn't seem to be any database backed models in the mobile module, I left this at the base level as a starting point.